lib/hem/tasks/magento2/createpatch.rb

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env ruby
+# ^ Syntax hint
+
+desc 'Create magento2 patch'
+task :createpatch do
+  package_name = Hem.ui.ask("which package did you modify? (e.g. module-catalog)")
+  patch_name = Hem.ui.ask("what should be the name of the patch? (e.g. my awesome fix )")
+  patch_tmp_path = '/tmp/patchtmp/'
```

Comment:
Would it be okay to do something like this? https://github.com/inviqa/hem/blob/431174d640b025ee965602d94303c113fabf976b/lib/hem/lib/vm/inspector.rb#L12